src/snippets/regional-collection.v1.yaml

```diff
@@ -0,0 +1,35 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Collection snippet
+type: object
+properties:
+    id:
+        $ref: ../misc/id.v1.yaml
+    title:
+        description: Title
+        type: string
+    impactStatement:
+        description: Description of the collection
+        type: string
+    published:
+        title: Publication date (UTC)
+        $ref: ../misc/date-time.v1.yaml
+    updated:
+        title: Updated date (UTC)
+        $ref: ../misc/date-time.v1.yaml
+    image:
+        type: object
+        properties:
+            thumbnail:
+                $ref: ../misc/image.v1.yaml
+        required:
+          - thumbnail
+    subjects:
+        description: Collection subjects
+        type: array
+        items:
+            $ref: ../snippets/subject.v1.yaml
```

Comment:
Is this needed?

Reply:
I will ask, if not needed now I may still suggest that we keep it here but not implement it in Journal CMS at this moment. I suspect that we may need to support a listings page at some point for these collections.

Reply:
Have confirmed that we should keep this in place to keep parity with collections but we may not expose this field at this moment in Journal CMS as the value will not be in use at this moment. It is an optional field in any case.